### PR TITLE
Update 8.4.md - Minor punctual and structural changes

### PR DESCRIPTION
**Pull request by MrSkribb**
Minor punctual and structural changes for better readability.

## Author checklist (to be completed by original Author)
- [x] Is this document a good fit for the Rocky project ?
- [ ] Is this a non-English contribution? 
- [ ] Title and Author MetaTags have been inserted into the document 
- [ ] If applicable, steps and instructions have been tested to work on a real system
- [x] Did you perform an initial self-review to fix basic typos and grammatical correctness

## Rocky Documentation checklist  (to be completed by Rocky team) 
- [ ] 1st Pass (Check that document is good fit for project and author checklist completed)
- [ ] 2nd Pass (Technical Review - check for technical correctness) 
- [ ] 3rd Pass (Basic Editorial Review)
- [ ] 4th Pass (Detailed Editorial Review and Peer Review)
- [ ] 5th Pass (Include document in TOC)
- [ ] Final pass/approval (Final Review)

### CODE REVIEW

File: docs/release_notes/8.4.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Brand new in Rocky Linux 8.4 are the following new module streams:
 
 * IPsec VPN provided by Libreswan now supports TCP encapsulation and security labels for IKEv2.
 
-* The scap-security-guide packages have been rebased to version 0.1.54, and OpenSCAP has been rebased to version 1.3.4. These updates provide substantial improvements, including: Improved memory management,
+* The scap-security-guide packages have been rebased to version 0.1.54, and OpenSCAP has been rebased to version 1.3.4. These updates provide substantial improvements, including improved memory management,
```
Comment (NeilHanlon) [inline docs/release_notes/8.4.md:54]: these are verbatim from upstream and we should not modify them, I think.

File: docs/release_notes/8.4.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ Brand new in Rocky Linux 8.4 are the following new module streams:
 
 * **Proactive compaction** regularly initiates memory compaction work **before** a request for allocation is made. Therefore, latency for specific memory allocation requests is lowered.
 
-* A new implementation of slab memory controller for the **control groups** technology is now available in Rocky Linux 8.4. The slab memory controller brings improvement in slab utilization, and enables to shift the memory accounting from the page level to the object level. As a result, you can observe a significant drop in the total kernel memory footprint and positive effects on memory fragmentation.
+* A new implementation of slab memory controller for the **control groups** technology is now available in Rocky Linux 8.4. The slab memory controller brings improvement in slab utilization and provides the functionality to shift the memory accounting from the page level to the object level. As a result, you can observe a significant drop in the total kernel memory footprint and positive effects on memory fragmentation.
```
Comment (NeilHanlon) [inline docs/release_notes/8.4.md:70]: same with these ones

File: docs/release_notes/8.4.md [hunk 3]
```diff
@@ -96,7 +96,7 @@ As with any release, there are sometimes bugs or issues found in the process of
 
 !!! note "A Note about Secure Boot"
     We know many of you depend on Secure Boot. It is a non-trivial process to get Secure Boot for a new OS. This process is underway and the [shim-review](https://github.com/rhboot/shim-review) process should begin very soon.
-    Rocky Linux 8.4 will initially be released without Secure Boot support enabled, however once the proper packages have been built and signed, a second set of ISOs will be released for the Rocky Linux 8.4 with Secure Boot support.
+    Rocky Linux 8.4 will initially be released without Secure Boot support enabled, however, once the proper packages have been built and signed, a second set of ISOs will be released for the Rocky Linux 8.4 with Secure Boot support.
```
Comment (NeilHanlon) [inline docs/release_notes/8.4.md:99]: I got some feedback that the wording was confusing, feel free to reword if you'd like!

**Comment by NeilHanlon** [comment]
thanks! I left a couple comments

File: docs/release_notes/8.4.md (hunk 3 above)
Comment (MrSkribb) [reply docs/release_notes/8.4.md:99]: I understand and agree with your statements, I do believe the documentation above should be reworded to this though. Seems to be easier to understand and more specific: 

Reworded:
> Rocky Linux version 8.4 will initially be released **without** Secure Boot support enabled by default. However, once the proper packages have been built and signed, another set of ISOs for Rocky Linux version 8.4 will be released **with** Secure Boot support available.

**Comment by MrSkribb** [comment]
> thanks! I left a couple comments

No worries, please view my response to the comment above. :)

**Pull request merged**